apis/apps/v1alpha1/apimanager_types.go

```diff
@@ -84,6 +84,8 @@ type APIManagerSpec struct {
 	PodDisruptionBudget *PodDisruptionBudgetSpec `json:"podDisruptionBudget,omitempty"`
 	// +optional
 	Monitoring *MonitoringSpec `json:"monitoring,omitempty"`
+	// +optional
+	PriorityClassName *string `json:"priorityClassName,omitempty"`
```

Comment:
on global configuration for all? What if a customer wants to grant priority to some components only??

Reply:
@eguzki , thank you for comment. Now I'm trying to add separate Priority classes for groups, but maybe need for each pod ... Could you please help , how better implement it. Currently i did it as following, but not sure it's good enough - https://github.com/3scale/3scale-operator/blob/e4a4eb54ae29f3715eb3d99efc8a0d18a69d6476/apis/apps/v1alpha1/apimanager_types.go#L88-L92

Reply:
@eguzki This is new commit - priorityClass per Pod ; example for system - just to ask if it's ok , if yes, I will continue with other pods  -  https://github.com/3scale/3scale-operator/pull/808/commits/f42c9a193313dbdb338c5ca64cd7e7a7d5e38128 . Thank you

Reply:
@eguzki , thank you for comment. I'm trying to do it for each pod now. Could you please look at latest commits. Thank you. 
cc @MStokluska